src/extension.ts

```diff
@@ -441,6 +465,19 @@ export async function activate(context: vscode.ExtensionContext) {
             );
         }
     );
+    const clueNewFile: vscode.Disposable = vscode.commands.registerCommand(
+        "deviceSimulatorExpress.clue.newFile",
+        () => {
+            telemetryAI.trackFeatureUsage(
+                TelemetryEventName.MICROBIT_COMMAND_NEW_FILE
+            );
+            telemetryAI.runWithLatencyMeasure(
+                openClueTemplateFile,
+
+                TelemetryEventName.MICROBIT_PERFORMANCE_NEW_FILE
+            );
```

Comment:
I think we should just make it run openClueTemplateFile() for now and have a task to add telemetry later instead of using microbit telemetry.